src/components/Card/Card.tsx

```diff
@@ -0,0 +1,45 @@
+import { Card as CardComponent, CardContent, Skeleton } from '@mui/material/';
+import Typography from '@mui/material/Typography';
+import { CardActionArea, useMediaQuery } from '@mui/material';
+import { useNavigate } from 'react-router-dom';
+import { mediaQueries } from '../DesignSystem/DesignSystem';
+import { CardProps } from '../../types/types';
+import extract from '../../utils/extract';
+
+export default function Card({ zaaktype, loading }: CardProps) {
+  const mobileScreen = useMediaQuery(mediaQueries.mobile);
+  const navigate = useNavigate();
+
+  const handleClick = () => {
+    navigate('/' + extract.uuid(zaaktype.url));
```

Comment:
I think it would make more sense to have a path `/zaaktype/<uuid>`, since later we can also include other paths (like for example `/informatieobjecttype/<uuid>`)